lib/m3u8/playlist_item.rb

```diff
@@ -34,8 +34,12 @@ def codecs
 
     def to_s
       validate
-      "#EXT-X-STREAM-INF:PROGRAM-ID=#{program_id},#{resolution_format}" +
-        %(CODECS="#{codecs}",BANDWIDTH=#{bitrate}\n#{playlist})
+
+      str = "#EXT-X-STREAM-INF:"
+      str += [program_id_format, resolution_format, codecs_format, bandwidth_format].compact.join(',')
```

Comment:
Line is too long. [102/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.